CHANGES.rst

```diff
@@ -241,6 +241,10 @@ API Changes
 
 - ``astropy.io.ascii``
 
+  - Calls to ascii.read() for fixed-width tables may now omit one of the keyword
```

Comment:
I think API changes is more for changes that are not backward compatible.  This is more of a New Feature than API Change.

Also (more importantly) this isn't a bug fix so it will go in the 1.1 release, not 1.0.2.